python-server.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Is this to avoid implicit reference to python 3 on the system?

If so, what would happen if the user's system has python 3 but not python 2?